<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter_sql_result">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="sa_query">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor" stdset="0">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>848</width>
            <height>162</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="topMargin">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QTabWidget" name="tab_result">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>입력변수</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tw_bind">
             <attribute name="horizontalHeaderHighlightSections">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>변수명</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>DB Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Comment</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Value</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>출력변수</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tw_output">
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Variable ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Comment</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>실행결과</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTableView" name="tw_sqlResult"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="status_layout"/>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="session_layout"/>
   </item>
  </layout>
  <action name="action_connectDialog">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon/res.qrc">
     <normaloff>:/db/db_connect.png</normaloff>:/db/db_connect.png</iconset>
   </property>
   <property name="text">
    <string>DB Connect</string>
   </property>
   <property name="toolTip">
    <string>DB Connect</string>
   </property>
  </action>
  <action name="action_queryTest">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon/res.qrc">
     <normaloff>:/case/play.png</normaloff>:/case/play.png</iconset>
   </property>
   <property name="text">
    <string>Query Test</string>
   </property>
   <property name="toolTip">
    <string>Query Test</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Enter, Ctrl+Return</string>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon/res.qrc"/>
 </resources>
 <connections/>
</ui>
